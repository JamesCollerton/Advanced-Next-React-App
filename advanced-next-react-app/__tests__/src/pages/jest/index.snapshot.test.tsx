import { render } from '@testing-library/react';
import Home from '../../../../src/pages/jest/index';
 
it('Home component snapshot test', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});