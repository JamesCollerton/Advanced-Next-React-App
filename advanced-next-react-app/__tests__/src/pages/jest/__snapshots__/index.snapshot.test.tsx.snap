// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component snapshot test 1`] = `
<div>
  <main
    class="main"
  >
    <h1>
      Here's our testing page!
    </h1>
  </main>
</div>
`;
